jbpm-wb-forms/jbpm-wb-forms-client/src/main/java/org/jbpm/workbench/forms/client/display/KieWorkbenchFormsStartProcessDisplayer.java

```diff
@@ -58,8 +58,7 @@ public IsWidget getFormWidget() {
     @Override
     public void startProcessFromDisplayer() {
         if (formRenderer.isValid()) {
-            service.call(getStartProcessRemoteCallback(),
-                         getUnexpectedErrorCallback()).startProcessFromRenderContext(
```

Comment:
@bxf12315 can _getUnexpectedErrorCallback_ method be completely removed?

Reply:
@cristianonicolai  I think, it is ok, In here , just have two class call getUnexpectedErrorCallback of AbstractStartProcessFormDisplayer.

Reply:
The compilation is failing now, please take another look.